prime-router/docs/playbooks/how-to-use-token-auth.md

```diff
@@ -69,3 +69,17 @@ Grab just the `access_token` value you got back from step 3, and use it here for
 ```
 curl -H "authorization:bearer ???" -H "client:waters"  -H "content-type:text/csv" --data-binary "@./junk/waters.csv" "http://localhost:7071/api/waters"
 ```
+
+**Bonus Steps**
+
+To avoid cutting and pasting by hand, this glorious string of unixy gibberish will request a signed 5minute access token (Same as STEP 2 above), and then paste it into an environment variable called `$TOK`:
+
+```
+export TOK=$(./prime sender reqtoken --private-key my-es-keypair.pem --scope simple_report.default.report --name simple_report.default |  grep access_token | python3 -c "import json,sys; print(json.load(sys.stdin)['access_token'])")
```

Comment:
this is cool... but it seems like we could have an arg you pass to the `sender reqtoken` command to parse out that access token for you :-)

Reply:
But this one line summarizes my entire career....

Reply:
Here's a thought or 3 about crypto as it relates to this (otherwise excellent) doc...
1. (I confess to being an RSA key-gen-aholic) but consider x509 certs
2. 8192-bit RSA keygen takes 9.2 seconds on a new Mac laptop (also- it's just one kilobyte)
3. At least one (popular) EC variant has been cracked in the field.  Meanwhile, the largest quantum machine is 127 bits.  So 8,192 RSA good for a while at least.
Recapping - 8192 RSA good.  x509 certs better.